test_conformance/extensions/cl_khr_command_buffer/basic_command_buffer.cpp

```diff
@@ -432,7 +432,7 @@ struct OutOfOrderTest : public BasicCommandBufferTest
     OutOfOrderTest(cl_device_id device, cl_context context,
                    cl_command_queue queue)
         : BasicCommandBufferTest(device, context, queue),
-          out_of_order_queue(nullptr), out_of_order_command_buffer(this),
+          out_of_order_command_buffer(this), out_of_order_queue(nullptr),
```

Comment:
Don't think we should be changing this member initialization order. `out_of_order_queue` is defined before `out_of_order_command_buffer` on lines 504-505, so initializing `out_of_order_command_buffer` first I suspect would trigger a `-Wreorder` warning.